pkgs/development/node-packages/node-packages.json

```diff
@@ -21,11 +21,11 @@
 , "@tailwindcss/typography"
 , "@uppy/companion"
 , "@vue/cli"
-, "@webassemblyjs/cli"
-, "@webassemblyjs/repl"
+, {"@webassemblyjs/cli": "1.11.1"}
+, {"@webassemblyjs/repl": "1.11.1"}
 , "@webassemblyjs/wasm-strip"
-, "@webassemblyjs/wasm-text-gen"
-, "@webassemblyjs/wast-refmt"
+, {"@webassemblyjs/wasm-text-gen": "1.11.1"}
+, {"@webassemblyjs/wast-refmt": "1.11.1"}
```

Comment:
I just wanted to add suggest to add a comment here but json...

Reply:
Maybe the commit description is the best place? At least then `git blame` on this file will show why it was added.